intel_extension_for_transformers/llm/runtime/graph/scripts/one_click_run.py

```diff
@@ -55,7 +55,7 @@ def main(args_in: Optional[List[str]] = None) -> None:
     parser.add_argument(
```

Comment:
how about rename one_click_run.py to run.py for simplicity?

Reply:
Great advice, updated!